frontend/src/theme/components.scss

```diff
@@ -13,3 +13,16 @@ button,
 input {
   font-family: 'Noto Sans';
 }
+.newRole {
```

Comment:
This class is being used inside HomeView. Can you put there, and just let here the general things, like el-input and h4?